umei-openapi.json

```diff
@@ -2141,7 +2141,7 @@
         },
         "additionalProperties": true
```

Comment:
Shouldn't we add a description to explain what a portfolio is ? Does it still group several assets ?